sdks/go/pkg/beam/core/graph/coder/coder.go

```diff
@@ -229,6 +245,7 @@ func IsWKV(c *Coder) bool {
 
 // NewWKV returns a WindowedValue coder for the window of KV elements.
 func NewWKV(components []*Coder, w *window.Window) *Coder {
+	checkCodersNotNil(components)
```

Comment:
Consider calling this function 'checkCoders' since it doesn't need to reveal its implementation details, and affords the possibility to verify other invariants of the coders.